src/scripts/parseCodePath.ts

```diff
@@ -36,84 +36,81 @@ export async function parse(xml: string) {
     const primaryCodePath = di.$.primaryCodePath;
     const paths: Record<string, CodePathInfo> = {};
 
-    // check if primaryCodePath exists
-    if (primaryCodePath) {
```

Comment:
The diff on this in browser for github is not very intuitive. The change to this file essentially just removes this `if` check.